python/aixexplainer/setup.py

```diff
@@ -33,7 +33,7 @@
     install_requires=[
         "kserve>=0.7.0",
         "argparse >= 1.4.0",
-        "aix360 >= 0.1.0",
+        "aix360 >= 0.2.0",
```

Comment:
needed so that we make sure we avoid version `0.1.0` which does not include the `lime` module used [here](https://github.com/kserve/kserve/blob/36888daa9b8139706dd7e226caaac1e2bd4dc567/python/aixexplainer/aixserver/model.py#L20).